test/spec/app_spec.rb

```diff
@@ -830,6 +788,150 @@
         elems = @driver.find_elements(:css => ".team-members__list > div")
         expect(elems.size).to be > 1
       }
+    end    
+
+    it "should redirect to team page if user asking to join a team is already a member", bin4:true do
+      page = Page.new(config: @config, driver: @driver)
+      page.go(@config['api_path'] + '/test/session?email=new'+@user_mail)
+      page = TeamsPage.new(config: @config, driver: @driver).load
+      @driver.navigate.to @config['self_url'] + "/"+@team1_slug+"/join"
+      sleep 3
+      @wait.until {
+        expect(@driver.current_url.eql? @config['self_url']+"/"+@team1_slug ).to be(true)
+      }    
+    end
+
+    it "should reject member to join team", bin4:true do
+      user = api_register_and_login_with_email
+      page = TeamsPage.new(config: @config, driver: @driver).load
+      page.ask_join_team(subdomain: @team1_slug)
+      @wait.until {
+        expect(@driver.find_element(:class, "message").nil?).to be(false)
+      }
+      api_logout
+      @driver.quit
+      @driver = new_driver(webdriver_url,browser_capabilities)
+      page = Page.new(config: @config, driver: @driver)
+      page.go(@config['api_path'] + '/test/session?email='+@user_mail)
+      #As the group creator, go to the members page and approve the joining request.
+      page = TeamsPage.new(config: @config, driver: @driver).load
+          .disapprove_join_team(subdomain: @team1_slug)
+      @wait.until {
+        expect(@driver.page_source.include?('Requests to join')).to be(false)
+      }      
+    end
+
+    #TO DO PROBLEMS WITH MENU
```

Comment:
`FIXME` not `TO DO`

Reply:
I fixed the problem in this version and didn't removed the comment. 
Now it is removed.